tools/walletextension/test/wallet_extension_test.go

```diff
@@ -18,9 +18,7 @@ import (
 const (
 	errFailedDecrypt = "could not decrypt bytes with viewing key"
 	dummyParams      = "dummyParams"
-	jsonKeyTopics    = "topics"
 	_hostWSPort      = integration.StartPortWalletExtensionUnitTest
-	_testOffset      = 100 // offset each test by a multiplier of the offset to avoid port colision. ie: 	hostPort := _hostWSPort + _testOffset*2
 )
```

Comment:
These offsets were never required as same test files run tests sequentially.
Removing them because they were making the ci flaky